c10/cuda/test/impl/CUDAAssertionsTest.cu

```diff
@@ -0,0 +1,658 @@
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <c10/cuda/CUDADeviceAssertion.h>
+#include <c10/cuda/CUDAException.h>
+#include <c10/cuda/CUDAFunctions.h>
+#include <c10/cuda/CUDAStream.h>
+
+#include <sys/wait.h>
```

Comment:
Is this a Linux specific test? As there are no `sys/wait.h` on Windows

Reply:
@malfet - Apologies; I can't believe I missed that. I've removed the header and am now retesting.